### PR TITLE
Enable one-time creation / multiple Jira reporting of LeVA Docs

### PR DESCRIPTION
**Pull request by hrcornejo**

### CODE REVIEW

File: test/groovy/org/ods/usecase/LeVADocumentUseCaseSpec.groovy [hunk 1]
```diff
@@ -820,16 +820,25 @@ class LeVADocumentUseCaseSpec extends SpecHelper {
...
         when:
         usecase.notifyJiraTrackingIssue(documentType, message)
 
         then:
         1 * jiraUseCase.jira.getIssuesForJQLQuery(jqlQuery) >> documentIssues
-
-        then:
-        e = thrown(RuntimeException)
-        e.message == "Error: Jira query returned 3 issues: '${jqlQuery}'."
+        3 * jiraUseCase.jira.appendCommentToIssue(_, message)
```
Comment (metmajer) [inline test/groovy/org/ods/usecase/LeVADocumentUseCaseSpec.groovy:953]: @hrcornejo Why 3? There is only one label involved, correct?

File: test/groovy/org/ods/usecase/LeVADocumentUseCaseSpec.groovy [hunk 2]
```diff
@@ -820,16 +820,25 @@ class LeVADocumentUseCaseSpec extends SpecHelper {
         then:
         def e = thrown(RuntimeException)
         e.message == "Error: Jira query returned 0 issues: '${jqlQuery}'."
+    }
+
+    def "notify LeVA document issue with query returning more than 1 issue"() {
```
Comment (metmajer) [inline test/groovy/org/ods/usecase/LeVADocumentUseCaseSpec.groovy]: @hrcornejo Name should be changed

File: test/groovy/org/ods/usecase/LeVADocumentUseCaseSpec.groovy [hunk 3]
```diff
@@ -820,16 +820,25 @@ class LeVADocumentUseCaseSpec extends SpecHelper {
...
+        def documentIssues = createJiraDocumentIssues()
 
         when:
         usecase.notifyJiraTrackingIssue(documentType, message)
 
         then:
         1 * jiraUseCase.jira.getIssuesForJQLQuery(jqlQuery) >> documentIssues
-
-        then:
-        e = thrown(RuntimeException)
```
Comment (metmajer) [inline test/groovy/org/ods/usecase/LeVADocumentUseCaseSpec.groovy:942]: @hrcornejo We should test the exception being thrown if there are 0 issues

File: test/groovy/org/ods/usecase/LeVADocumentUseCaseSpec.groovy (hunk 1 above)
Comment (hrcornejo) [reply test/groovy/org/ods/usecase/LeVADocumentUseCaseSpec.groovy:953]: Nop, in this test there are 3 issues created, so 3 calls:
`    static List createJiraDocumentIssues() {
        def result = []

        result << createJiraIssue("1", "my-doc-A", "Document A")
        result << createJiraIssue("2", "my-doc-B", "Document B")
        result << createJiraIssue("3", "my-doc-C", "Document C")

        return result
    }`

File: test/groovy/org/ods/usecase/LeVADocumentUseCaseSpec.groovy (hunk 3 above)
Comment (hrcornejo) [reply test/groovy/org/ods/usecase/LeVADocumentUseCaseSpec.groovy:942]: That is done in the other method

File: test/groovy/org/ods/usecase/LeVADocumentUseCaseSpec.groovy (hunk 2 above)
Comment (hrcornejo) [reply test/groovy/org/ods/usecase/LeVADocumentUseCaseSpec.groovy]: Modify per: "notify LeVA document issue with query returning 3 issues"

**Comment by hrcornejo** [comment]
@metmajer ready

**Pull request merged**